### PR TITLE
Add tests for WebDriver BiDi browsingContext.captureScreenshot

### PR DESCRIPTION
**Pull request by jgraham**

### CODE REVIEW

**Comment by whimboo** [comment]
Hi @sadym-chromium. We would appreciate a review from your side before we land this patch in mozilla-central. I know that you haven't added support for screenshots yet, so maybe just skim through? Thanks!

**Comment by sadym-chromium** [comment]
on it

File: webdriver/tests/bidi/browsing_context/capture_screenshot/invalid.py.ini [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[invalid.py]
+  [test_closed_frame]
+    bug: 1761445
+    expected: FAIL
```
Comment (sadym-chromium) [inline webdriver/tests/bidi/browsing_context/capture_screenshot/invalid.py.ini:4]: It seems to reference your internal bug tracker

File: webdriver/tests/support/fixtures_bidi.py [hunk 2]
```diff
@@ -54,3 +55,39 @@ async def _():
...
+                iframe.src = url;
+                document.documentElement.lastElementChild.append(iframe);
+                return new Promise(resolve => iframe.onload = () => resolve(id))
+            }""",
+            target={"context": context["context"]},
+            await_promise=True)
+        iframe_dom_id = resp["value"]
+
+        all_contexts = await bidi_session.browsing_context.get_tree(root=context["context"])
+        frame_id = all_contexts[0]["children"][-1]["context"]
```
Comment (sadym-chromium) [inline webdriver/tests/support/fixtures_bidi.py:85]: NIT: not sure if the spec guarantees the order of the frames

**Review by sadym-chromium — APPROVED** [review]
I cannot run it yet in Chrome, but looks reasonable.

**Comment by whimboo** [comment]
This PR got finally merged via https://github.com/web-platform-tests/wpt/pull/37454.

**Pull request closed without merging**